subsys/bluetooth/host/smp.c

```diff
@@ -3875,8 +3889,15 @@ static uint8_t smp_ident_addr_info(struct bt_smp *smp, struct net_buf *buf)
 			 * present before ie. due to re-pairing.
 			 */
 			if (!bt_addr_le_is_identity(&conn->le.dst)) {
+				struct addr_match addr_match = {
+					.rpa = &conn->le.dst,
+					.id_addr = &req->addr,
+				};
+
+				bt_conn_foreach(BT_CONN_TYPE_ALL,
```

Comment:
can you change that to `BT_CONN_TYPE_LE`? Otherwise you access the wrong union variant.

Reply:
thanks, it is fixed now